avocado/utils/asset.py

```diff
@@ -361,13 +367,15 @@ def get_metadata(self):
         :return: metadata
         :rtype: dict or None
         """
-        asset_file = self.find_asset_file()
-
-        if asset_file is not None:
-            basename = os.path.splitext(asset_file)[0]
-            metadata_file = "%s_metadata.json" % basename
-            if os.path.isfile(metadata_file):
-                with open(metadata_file, "r") as f:
-                    metadata = json.load(f)
-                    return metadata
-        return None
```

Comment:
There is a possible improvement here to change the behavior of get_metadata and throw an exception when the metadata is not available to follow the pattern from the class, but this is not related to the objective of this PR, so I'm taking note to change it as soon as #3600 is closed